src/rdo.rs

```diff
@@ -2220,7 +2220,7 @@ pub fn rdo_loop_decision<T: Pixel>(
                       pli,
                     );
                     rate += if fi.sequence.enable_restoration {
-                      cw.count_lrf_switchable(
+                      cw.fc.count_lrf_switchable(
```

Comment:
so this shouldn't be needed anymore.

Reply:
???
```
error[E0502]: cannot borrow `*cw` as immutable because it is also borrowed as mutable
    --> src/rdo.rs:2223:23
     |
2041 |   let mut tileblocks_subset = cw.bc.blocks.subregion_mut(
     |                               ------------ mutable borrow occurs here
...
2223 |                       cw.count_lrf_switchable(
     |                       ^^ immutable borrow occurs here
...
2317 |             tileblocks_subset.set_cdef(loop_sbo, best_new_index as u8);
     |             ----------------- mutable borrow later used here
```

Reply:
It can wait later. let's land it.